### PR TITLE
Make `--verbose` param behave as per documentation

### PR DESCRIPTION
**Pull request by mirgee**
Addressing #1739, this PR makes the `--verbose` param behave as per documentation: when it is specified as flag, i.e.  `--verbose`,  debug level is set for `lbry` logger as default, and when a list of strings is passed, debug level is set to each as a logger. This does not check that the loggers are valid modules.

### CODE REVIEW

**Comment by tzarebczan** [comment]
Thanks for the PR @mirgee ! We'll get a review on it soon. 

As usual, can we send you some [appreciation](https://lbry.com/faq/appreciation) for the contribution?

File: lbry/lbry/extras/cli.py [hunk 1]
```diff
@@ -242,11 +242,11 @@ def run_daemon(args: list, conf: Config):
...
 
-    if args.verbose:
-        log.setLevel(logging.DEBUG)
+    log.setLevel(logging.DEBUG if args.verbose == [] else logging.INFO)
+    for module in args.verbose or []:
+        logging.getLogger(module).setLevel(logging.DEBUG)
+    if isinstance(args.verbose, list):
         loop.set_debug(True)
-    else:
-        log.setLevel(logging.INFO)
```
Comment (eukreign) [inline lbry/lbry/extras/cli.py]: The conditionals and logic here is a bit convoluted and hard to follow, can you simplify/clean this up?

Comment (mirgee) [reply lbry/lbry/extras/cli.py]: Hopefully cleared it up. I can add comments or a few new variables, if you think it's necessary...

File: lbry/lbry/extras/cli.py [hunk 2]
```diff
@@ -242,11 +242,12 @@ def run_daemon(args: list, conf: Config):
...
-    if args.verbose:
-        log.setLevel(logging.DEBUG)
+    modules = args.verbose
+    if modules is not None:
         loop.set_debug(True)
-    else:
-        log.setLevel(logging.INFO)
+        log.setLevel(logging.DEBUG if modules == [] else logging.INFO)
+        for module in modules:
+            logging.getLogger(module).setLevel(logging.DEBUG)
```
Comment (eukreign) [inline lbry/lbry/extras/cli.py]: i think it's still convoluted, also adding more variables usually makes things more complicated (and if you need to add comments then it's even more complicated :-p)

what about (no additional variables and no inline `if` statement):
```
    if args.verbose is not None:
        loop.set_debug(True)
        if len(args.verbose) > 0:
            log.setLevel(logging.INFO)
            for module in args.verbose:
                logging.getLogger(module).setLevel(logging.DEBUG)
        else:
            log.setLevel(logging.DEBUG)
```

Comment (mirgee) [reply lbry/lbry/extras/cli.py]: I agree - it's that I didn't know why it was not readable and thought naming things would help. This could work.

**Comment by eukreign** [comment]
@mirgee can you rebase this and we'll merge
